crates/solana-event-forwarder/src/component.rs

```diff
@@ -348,182 +239,18 @@ fn map_gateway_event_to_amplifier_event(
                     )
                     .build(),
             );
-            tracing::info!(message_id = ?approved_message.message_id, "Message approved");
+            tracing::info!("message approved");
             Some(amplifier_event)
         }
-        MessageExecuted(ref _executed_message) => {
+        GatewayEvent::MessageExecuted(ref _executed_message) => {
```

Comment:
This is the part for https://github.com/eigerco/solana-axelar-internal/issues/572 issue to be implemented